docs/theory/diffusion/fitting.rst

```diff
@@ -0,0 +1,239 @@
+Fitting Methods
```

Comment:
I find `Fitting Methods` a bit of an odd section header, since the stuff described in `Ensemble estimates` is also just a fitting method.

I feel that `Mean Squared Displacements`, `TAMSD`, `Ensemble MSD` and `Important Takeaways` are all pure background about MSDs and should probably be under a header `Mean Squared Displacements`.

Then the next three (OLS, GLS, CVE) are estimators that could have their own section.